tests/test_grpo_trainer.py

```diff
@@ -49,7 +49,7 @@ def test_training(self, config_name):
             training_args = GRPOConfig(
                 output_dir=tmp_dir,
                 learning_rate=0.1,  # increase the learning rate to speed up the test
-                per_device_train_batch_size=2,  # reduce the batch size to reduce memory usage
+                per_device_train_batch_size=3,  # reduce the batch size to reduce memory usage
```

Comment:
New requirement: The global batch size must be evenly divisible by the number of generations per prompt.